test/unit/slot/fallback-content.js

```diff
@@ -6,6 +7,11 @@ describe('fallback-content', () => {
   let slot;
   let fallback;
 
+  // We don't need to run these tests at all against v0.
```

Comment:
Then we'd be testing chrome native. If there are tests missing for the polyfill, these should go into the slot polyfill tests and only be tested against the polyfill.